### PR TITLE
Testsuite with CMake: get Back the Blue Squares

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes
Add a python script to parse the Installation cmake logs and retrieve the information concerning the un-configured packages, and put a blue square in the result table like there usually is. This script creates the missing log files so that there is something to show when clicking on the blue square, usually the "NOTICE: missing deps " logged by the cmake configuration.

There is an example of the result [here](https://cgal.geometryfactory.com/~mgimeno/test_suite/TESTRESULTS/results-5.0-Ic-190403.shtml). 

Most of the blue squares are simply untested packages, but AABB_tree_Demo is ignored because of missing Qt5, and clicking on the square shows just that.

## Release Management

* Affected package(s):Testsuite and Developer Scripts

### CODE REVIEW

**Comment by maxGimeno** [comment]
Tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-5.shtml

**Comment by lrineau** [comment]
[**@maxGimeno**](https://github.com/maxGimeno) commented on [Apr 8, 2019, 3:48 PM GMT+2](https://github.com/CGAL/cgal/pull/3832#issuecomment-480839180 "2019-04-08T13:48:36Z - Replied by Github Reply Comments"):
> Tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-5.shtml

The column `Debian-stable-arm-linux-gnu-32bits` is fully blue. Are you sure this PR works correctly?

**Comment by maxGimeno** [comment]
I did not see that... I gues it doesn't work at all then -_-

**Comment by maxGimeno** [comment]
TODO: In the reports of a ctest testsuite, the list of the merged PRs is missing.

**Comment by lrineau** [comment]
I think `run_testsuite_with_cmake` should be renamed `run_testsuite_with_ctest`, to be consistent with the naming of `autotest_cgal_with_ctest`.

File: Scripts/developer_scripts/run_testsuite_with_cmake [hunk 1]
```diff
@@ -319,9 +325,14 @@ run_test_on_platform()
   TEST_REPORT="TestReport_${CGAL_TESTER}_${PLATFORM}"
   mkdir -p Installation
   chmod 777 Installation
-  cp "${CGAL_BINARY_DIR}/package_installation.log" "Installation/${TEST_REPORT}"
+  cat "${CGAL_BINARY_DIR}/package_installation.log" >> "Installation/${TEST_REPORT}"
   
+  #call the python script to complete the results report.
+  python3 ${CGAL_DIR}/${TESTSUITE_DIR}test/get_deactivated_packages_info.py Installation/${TEST_REPORT} ${TEST_REPORT} results_${CGAL_TESTER}_${PLATFORM}.txt
```
Comment (lrineau) [inline Scripts/developer_scripts/run_testsuite_with_cmake]: I do not understand the name of the script: `get_deactivated_packages_info.py`. Now that the script is finished, you can probably find a better name for it.

**Comment by maxGimeno** [comment]
Successfully testsed in [Ic-29](https://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-29.shtml)

**Pull request merged**